Headers/DebugServer2/Architecture/X86/RegistersDescriptors.h

```diff
@@ -179,22 +179,22 @@ enum /* lldb_reg */ {
   reg_lldb_fop = 39,
   reg_lldb_mxcsr = 40,
   reg_lldb_mxcsrmask = 41,
-  reg_lldb_ymm0 = 50,
-  reg_lldb_ymm1 = 51,
-  reg_lldb_ymm2 = 52,
-  reg_lldb_ymm3 = 53,
-  reg_lldb_ymm4 = 54,
-  reg_lldb_ymm5 = 55,
-  reg_lldb_ymm6 = 56,
-  reg_lldb_ymm7 = 57,
-  reg_lldb_xmm0 = 58,
-  reg_lldb_xmm1 = 59,
-  reg_lldb_xmm2 = 60,
-  reg_lldb_xmm3 = 61,
-  reg_lldb_xmm4 = 62,
-  reg_lldb_xmm5 = 63,
-  reg_lldb_xmm6 = 64,
-  reg_lldb_xmm7 = 65,
+  reg_lldb_ymm0 = 58,
```

Comment:
I'm sure it's not bad, unclear whether it's good. But this is an auto-generated change, coming from the register set regrouping.